maps/chronosphere/event_functions.lua

```diff
@@ -312,8 +325,8 @@ end
 function Public_event.on_technology_effects_reset(event)
 	local objective = Chrono_table.get_table()
 	if event.force.name == "player" then
-		game.forces.player.character_inventory_slots_bonus = game.forces.player.character_inventory_slots_bonus + objective.invupgradetier * 10
-		game.forces.player.character_loot_pickup_distance_bonus = game.forces.player.character_loot_pickup_distance_bonus + objective.pickupupgradetier
+		game.forces.player.character_inventory_slots_bonus = objective.invupgradetier * 10 --fixed typo?
+		game.forces.player.character_loot_pickup_distance_bonus = objective.pickupupgradetier --fixed typo?

```

Comment:
Same here, the previous state needs to be taken into account.

Reply:
Yep.